### ISSUE TITLE
prekallikrein deficiency

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7650]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Reduced concentration of prekallikrein (PK; also known as Fletcher factor) in the blood circulation. PK is the precursor of plasma kallikrein, a serine protease that activates kinins. PK is cleaved to produce kallikrein by activated Factor XII.

Comment: Most cases of prekallikrein deficiency are asymptomatic.
PMID: 15461630
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
612423 
PMID: 15461630  1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
